backend/OR-Tools/OR-Tools/ORSolver.cs

```diff
@@ -36,6 +37,7 @@ public void Solve()
 
             // Solve the problem
             solution = routing.SolveWithParameters(searchParameters);
+            System.Console.WriteLine("Solved");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4618867)